### PR TITLE
log4j news

### PR DESCRIPTION
**Pull request by browkin**
- Added Log4j News article
- EN & DE

---
Internal Tracking ID: [EXPOSUREAPP-11059](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11059)

### CODE REVIEW

**Review by dsarkar — APPROVED** [review]

File: src/data/faq.json [hunk 1]
```diff
@@ -91,6 +91,17 @@
...
+                    {
+                        "title": "Statement regarding the Log4J vulnerability",
+                        "anchor": "log4j",
+                        "active": true,
+                        "textblock": [
+                            "Statement regarding the Log4J vulnerability.",
+                            "Log4j is a widely used Java-based utility for logging error messages in applications. The Log4j library is widely used in a very broad range of enterprise software and services from many major vendors. A vulnerability related to the Log4j library was discovered recently and allows remote code execution.",
+                            "In response to this recent discovery and in order to minimize potential impacts the project teams of SAP and TSI join the worldwide effort and have taken action. This includes amongst others a version update to 2.16.0",
+                            "There is a series of pull requests related to this issue. They can be found in the repository of our open, source project: <a href='https://github.com/corona-warn-app/cwa-server/pull' target='_blank' rel='noopener'>https://github.com/corona-warn-app/cwa-server/pull</a>"
```
Comment (Ein-Tim) [inline src/data/faq.json:102]: Remove ","

```suggestion
                            "There is a series of pull requests related to this issue. They can be found in the repository of our open source project: <a href='https://github.com/corona-warn-app/cwa-server/pull' target='_blank' rel='noopener'>https://github.com/corona-warn-app/cwa-server/pull</a>"
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Remove "," too much

**Comment by dsarkar** [comment]
@Ein-Tim Thanks.

**Comment by MikeMcC399** [comment]
@dsarkar 
I wonder why this is going into the [news section of the FAQs](https://www.coronawarn.app/en/faq/#news).

For many months all news articles have gone into the [blog ](https://www.coronawarn.app/en/blog/) instead.

What is the reason to use the FAQs instead of the blog?

**Comment by Ein-Tim** [comment]
@MikeMcC399 

https://www.coronawarn.app/de/faq/#google_security_vulnerability was also was published in the FAQ through #1175.

File: src/data/faq.json [hunk 2]
```diff
@@ -91,6 +91,17 @@
...
                 "id": "news",
                 "accordion": [
+                    {
+                        "title": "Statement regarding the Log4J vulnerability",
+                        "anchor": "log4j",
+                        "active": true,
+                        "textblock": [
+                            "Statement regarding the Log4J vulnerability.",
+                            "Log4j is a widely used Java-based utility for logging error messages in applications. The Log4j library is widely used in a very broad range of enterprise software and services from many major vendors. A vulnerability related to the Log4j library was discovered recently and allows remote code execution.",
+                            "In response to this recent discovery and in order to minimize potential impacts the project teams of SAP and TSI join the worldwide effort and have taken action. This includes amongst others a version update to 2.16.0",
```
Comment (MikeMcC399) [inline src/data/faq.json:101]: Change of grammar and specific mention of server (not app).

```suggestion
                            "In response to this recent discovery and in order to minimize potential impacts the project teams of SAP and TSI have joined the worldwide effort and taken action. This includes amongst others a server version update to 2.16.0",
```

File: src/data/faq_de.json [hunk 3]
```diff
@@ -91,6 +91,17 @@
...
                 "id": "news",
                 "accordion": [
+                    {
+                        "title": "Erklärung zur Sicherheitslücke in Log4J",
+                        "anchor": "log4j",
+                        "active": true,
+                        "textblock": [
+                            "Log4j ist ein weit verbreitetes Java-basiertes Werkzeug zur Protokollierung von Fehlermeldungen in Anwendungen.",
+                            "Die Log4j-Bibliothek wird in einem breiten Spektrum von Unternehmenssoftware und in Diensten vieler großer Anbieter eingesetzt. Kürzlich wurde eine Sicherheitslücke im Zusammenhang mit der Log4j-Bibliothek entdeckt, die die Ausführung von Remote-Code ermöglicht.",
+                            "Als Reaktion auf diese jüngste Entdeckung und um mögliche Auswirkungen zu minimieren, haben sich die Projektteams von SAP und TSI den weltweiten Bemühungen angeschlossen und Maßnahmen ergriffen. Dazu gehört unter anderem ein Versionsupdate auf 2.16.0 ",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:101]: Suggest to explicitly state this is about server software, not the app software.

```suggestion
                            "Als Reaktion auf diese jüngste Entdeckung und um mögliche Auswirkungen zu minimieren, haben sich die Projektteams von SAP und TSI den weltweiten Bemühungen angeschlossen und Maßnahmen ergriffen. Dazu gehört unter anderem ein Server-Versionsupdate auf 2.16.0 ",
```

Comment (Ein-Tim) [reply src/data/faq_de.json]: Mhh, this sounds like the server was updated, but in fact, only log4j was updated. But this is just a minor detail.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I added some inline suggestions for change, particularly to state that this is a server change.

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for the review. 

> I wonder why this is going into the news section of the FAQs.
> 
> For many months all news articles have gone into the blog instead.
> 
> What is the reason to use the FAQs instead of the blog?

Both FAQ and Blog options had been discussed, however, it was decided to use the FAQ.

**Comment by MikeMcC399** [comment]
@dsarkar 
> Both FAQ and Blog options had been discussed, however, it was decided to use the FAQ.
Thanks for letting me know that the options were considered.

You might also want to consider what happens to articles posted to the FAQ news section when they are no longer news because they were posted a long time in the past. It seems inconsistent to move some articles and leave others in the original posting position.

**Comment by MikeMcC399** [comment]
@dsarkar / @browkin 

In the English version only https://www.coronawarn.app/en/faq/#log4j the title "Statement regarding the Log4J vulnerability" is repeated in the text, so this is redundant.

Do you want to correct this?

**Pull request merged**